calour/heatmap/heatmap.py

```diff
@@ -89,46 +89,53 @@ def _truncate_middle(x, length=16):
         return ['%s..%s' % (i[:mid], i[-mid:]) if len(i) > length else i for i in y]
 
 
-def _set_axis_ticks(ax, which, ticklabels, tickmax, n, kwargs, ticklabel_len):
+def _set_axis_ticks(ax, which, ticklabels, tickmax, n, kwargs, ticklabel_len, transition=True):
     '''Plot tick and ticklabels of x or y axis.
 
     Parameters
     ----------
     ax : matplotlib.axes.Axes
         The axes to plot into
-    Parameters mean axes, x or y axis, list of tick labels, max number
+    Parameters: axes, x or y axis, list of tick labels, max number
     of ticks, the upper bound of xlim/ylim, dict passing as text
     property, the length of each tick label.
 
```

Comment:
can you add a description of transition parameter?

Reply:
added